src/benchmark/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ add_executable(
     operators/union_all_benchmark.cpp
     table_generator.cpp
     table_generator.hpp
+    tpch_db_generator_benchmark.cpp
 )
```

Comment:
@mrks regarding doc'ing the hana stuff: I've added some semantic docs, but I can't really explain the (very basic) hana things I'm doing here. Explaining hana is a job for the hana docs, imo

Reply:
👍

Reply:
This is what I was thinking off.